packages/compass-components/src/components/signal-popover.tsx

```diff
@@ -69,6 +74,7 @@ const signalCardContentDarkModeStyles = css({
 });
 
 const signalCardTitleStyles = css({
+  maxWidth: '90%',
```

Comment:
Percentages here don't look right, is this for the cancel button? If that's the case you probably want to use `calc(100% - <space needed for button>)` to figure out exactly the max width that we can have here (or a padding right, which is how we handle it somewhere else I think because this button is not in our control)